### ISSUE TITLE
Loss Graph spike when epoch starting point

### ISSUE THREAD

**Issue by cxz1418** [deepinsight/insightface#1704]
Hello~ I'm learning with my custom dataset with arcface_torch code but,

Even though it is not an LR adjustment interval, the loss drops significantly 
and then rises at the start of the Epoch. What's the problem?

<img width="878" alt="스크린샷 2021-08-11 오후 5 44 36" src="https://user-images.githubusercontent.com/11434363/128998805-85a0fcdb-4b51-4e80-8903-11a2dcc26f95.png">

<img width="874" alt="스크린샷 2021-08-11 오후 5 45 30" src="https://user-images.githubusercontent.com/11434363/128998826-eb0efd1a-d072-4330-820e-5566216fe319.png">

**Comment by ine1113** [comment]
I'm also experiencing this issue, but with vanilla vpl training script. The loss drops slightly at the moment of epoch change, and slowly climbs back to the previous range. 

Any help would be appreciated.
